.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: scala
 scala:
   - 2.11.7
+jdk:
+  - oraclejdk8
```

Comment:
This is the travis build info, this should not be brought into the rally branch, or used by rally in any way